`swift package init` doesn't generate `main.swift` any more

```
$ swift package init
Creating library package: testproject
Creating Package.swift
Creating .gitignore
Creating Sources/
Creating Sources/testproject.swift
Creating Tests/
Creating Tests/LinuxMain.swift
Creating Tests/testproject/
Creating Tests/testproject/testprojectTests.swift
```

This means that when you follow the README you need to create `main.swift` yourself, otherwise if you add the code to `Sources/testproject.swift` you see:

```
testproject/Sources/testproject.swift:4:1: error: expressions are not allowed at the top level
HeliumLogger.use()
^
testproject/Sources/testproject.swift:8:1: error: expressions are not allowed at the top level
router.get("/") {
^
testproject/Sources/testproject.swift:14:1: error: expressions are not allowed at the top level
Kitura.addHTTPServer(onPort: 8090, with: router)
^
testproject/Sources/testproject.swift:15:1: error: expressions are not allowed at the top level
Kitura.run()
^
```

--- a/README.md
+++ b/README.md
@@ -147,7 +147,7 @@ Let's develop your first Kitura web application!
 
   ```
   $ cd myFirstProject
-  $ swift package init
+  $ swift package init --type executable
   ```
 
   Now your directory structure under myFirstProject should look like this:

